### ISSUE TITLE
Buy tag in store instead of e.g. every individual wood

### ISSUE THREAD

**Issue by TCastelein** [StrangeLoopGames/EcoSuggestions#1670]
TLDR; Use tag in store buy part to denote a group of items. For instance lumber tag that can be filled by 3 items. To avoid overflowing your stockpiles.

With the new 9.5? update we can split the bought products and sell-able products for a store (among other things). I use this to make sure I can exactly say how much of something I want and then make sure there is always space for it. Because there is nothing worse then driving somewhere and getting the message their storage is full and you cannot sell anything.

However some items have different types which means i have to split them up even though I only want a combined of lets say 100 lumber. Right now I'm a mechanic and I only care about the lumber tag it does not matter if it is softwood, hardwood or regular lumber  they are all equal as I only need the lumber tag for my recipes. And now I only buy the regular lumber, but someone in the server has loads of softwood lumber ready to go. There is a mismatch here.

To fix this issue my suggestion is to allow for tags to be available in the store. This means I can buy exactly 100 lumber in 1 buy order: 100 lumber tag. And don't have to do 33 hardwood lumber, 33 lumber and 34 softwood lumber so I don't overflow my stockpile. The tag can then be filled in any order like 50 lumber 50 hardwood lumber or 99 softwood lumber 1 lumber. Of course this will take up more space then 100 lumber but that is something I can work with.

This also works in other area's like wood. If all you want to do is make charcoal or any type of hewn log it would be very nice to just setup a buy order of 1 stockpile of wood tag and be done with it.

I do believe it only works in the buy section of the store as selling will make it a bit more complicated as you do not know what you will get.

**Comment by 1ntrospect1on** [comment]
+1 on tags being able to be listed in store. Shouldnt be hard to implement.

It's so annoying when you have to set a buy order for both softwood hewn log, hardwood hewn log, hewn logs, and then all different kinds of spruce logs, redwood logs, etc because you dont know what people will sell you... 
You'll never end up getting the actual amount of wood you need.

**Comment by joamla96** [comment]
+1 on this one from me as well
